src/components/MessageInput/StopGeneratingButton.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+export type StopGeneratingButtonProps = {
+  onClick: () => void;
```

Comment:
I would say that it is not necessary to declare onClick apart from other native button propse. I think we can keep here something line

```
export type StopGeneratingButtonProps = React.ComponentProps<'button'>;
```